policy/modules/services/mta.te

```diff
@@ -285,7 +285,12 @@ optional_policy(`
 	userdom_dontaudit_use_user_ptys(system_mail_t)
 
 	optional_policy(`
+ifdef(`hide_broken_symptoms',`
```

Comment:
Please change this to `distro_debian`. I'd rather not add "broken_symptoms" blocks, but instead elimitate them since they are essentially always enabled.

Reply:
ok